src/templates/zh-CN/2019/methodology.html

```diff
@@ -74,11 +74,11 @@ <h2 id="dataset"><a href="#dataset" class="anchor-link">关于数据集</a></h2>
         </p>
 
         <p>
-          Web Almanac中列出的所有指标都可以通过BigQuery上的数据集公开重现。您可以在我们的<a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2019">GitHub 仓库</a>浏览我们的所有章节使用的查询 .
+          Web Almanac中列出的所有指标都可以通过BigQuery上的数据集公开重现。您可以在我们的<a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2019">GitHub 仓库</a>浏览我们的所有章节使用的查询 。
         </p>
 
         <p class="note">
-          请注意，其中一些查询相当大，如果你自己运行可能比较 <a href="https://cloud.google.com/bigquery/pricing">昂贵</a> , 因为BigQuery是按TB收费. 为了帮助控制你的开支, 请看 Tim Kadlec的文章 <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">在不破产的情况下来使用BigQuery</a>.
+          请注意，其中一些查询相当大，如果你自己运行可能比较 <a href="https://cloud.google.com/bigquery/pricing">昂贵</a> , 因为BigQuery是按TB收费。为了帮助控制你的开支, 请看 Tim Kadlec的文章 <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">在不破产的情况下来使用BigQuery</a>。
```

Comment:
Unnecessary spaces? And those the right commas to use?

Reply:
spaces deleted